docs/tutorial/common_hpc_dependencies.rst

```diff
@@ -0,0 +1,173 @@
+.. # Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level LICENSE file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _CommonHPCDependencies:
+
+Common HPC Dependencies
+=======================
+
+BLT creates named targets for the common HPC dependencies that most HPC projects need, such as
+MPI, CUDA, HIP, and OpenMP.  As previously mentioned in :ref:`AddingTests`, BLT also provides 
```

Comment:
Should we mention that BLT simplifies getting these all to work together, e.g. `mpi` + `cuda` + `openmp`